src/endpoints/ECatCourse/ECatAssignmentGroup.ts

```diff
@@ -107,9 +113,36 @@ class ECatAssignmentGroup extends EndpointCategory {
       courseId?: number,
       name?: string,
       weight?: number,
+      dropLowest?: number,
+      dropHighest?: number,
+      neverDrop?: number[],
     },
     config?: APIConfig,
   ): Promise<CanvasAssignmentGroup> {
+    // Create rules
+    const {
+      dropLowest,
+      dropHighest,
+      neverDrop,
+    } = opts;
+    let rules: { [k: string]: (number | number[]) };
+    if (
+      (dropLowest && dropLowest > 0)
+      || (dropHighest && dropHighest > 0)
+    ) {
+      rules = {};
+      if (dropLowest) {
+        rules.drop_lowest = dropLowest;
+      }
+      if (dropHighest) {
+        rules.drop_highest = dropHighest;
+      }
+      if (neverDrop) {
```

Comment:
This could still trigger when `neverDrop=[]` i.e. when it's an empty array. Just double-checking that that's intended behavior.